bin/cliain/src/keys.rs

```diff
@@ -1,10 +1,12 @@
-use aleph_client::{staking_bond, set_keys, rotate_keys, create_connection, KeyPair};
+use aleph_client::{create_connection, rotate_keys, set_keys, staking_bond, KeyPair};
 use primitives::staking::MIN_VALIDATOR_BOND;
 use substrate_api_client::XtStatus;
 
-pub fn prepare(node: String, key: KeyPair) {
-    let connection = create_connection(&node).set_signer(key.clone());
+pub fn prepare(node: String, ssl: bool, key: KeyPair) {
+    let connection = create_connection(&node, ssl).set_signer(key.clone());
```

Comment:
Is it possible to make a `connection` object not in the command functions, but in `main` of `cliain` create? And pass the `connection` object to those methods. Btw this PR adds way more command: https://github.com/Cardinal-Cryptography/aleph-node/pull/338 , so I'd like to establish a common convention